dist/three.ar.js

```diff
@@ -1135,6 +1135,73 @@ exports.default = ARReticle;
 "use strict";
 
 
+var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) { return typeof obj; } : function (obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj; };
+
+(function () {
+  if (window.webarSpeechRecognitionInstance) {
+    var addEventHandlingToObject = function addEventHandlingToObject(object) {
+      object.listeners = {};
+      object.addEventListener = function (eventType, callback) {
+        if (!callback) return this;
+        var listeners = this.listeners[eventType];
+        if (!listeners) {
+          listeners = [];
+          this.listeners[eventType] = listeners;
+        }
+        if (listeners.indexOf(callback) < 0) {
+          listeners.push(callback);
+        }
+        return this;
+      };
+      object.removeEventListener = function (eventType, callback) {
+        if (!callback) return this;
+        var listeners = this.listeners[eventType];
+        if (listeners) {
+          var i = listeners.indexOf(callback);
+          if (i >= 0) {
+            this.listeners[eventType] = listeners.splice(i, 1);
+          }
+        }
+        return this;
+      };
+      object.callEventListeners = function (eventType, event) {
+        if (!event) event = { target: this };
+        if (!event.target) event.target = this;
+        event.type = eventType;
+        var onEventType = 'on' + eventType;
+        if (typeof this[onEventType] === 'function') {
+          this[onEventType](event);
+        }
+        var listeners = this.listeners[eventType];
+        if (listeners) {
+          for (var i = 0; i < listeners.length; i++) {
+            var typeofListener = _typeof(listeners[i]);
+            if (typeofListener === "object") {
```

Comment:
single quotes for strings in javascript is the prefered style. Also, consistency is important and you already used single quotes for 'on' above.